src/dev/flang/be/c/CTypes.java

```diff
@@ -43,6 +43,21 @@ public class CTypes extends ANY
 {
 
 
+  /*----------------------------  constants  ----------------------------*/
+
+
+  /*
+   * type of clazzId field in ref instances
+   */
+  private static final String typeClazzID = scalar(SpecialClazzes.c_u32);
+
+
+  /*
+   * type of the tag field in choices
+   */
+  private static final String typeTagName = scalar(SpecialClazzes.c_i32);
```

Comment:
These should IMHO be just `"uint32_t"` or `"int32_t"` since the type is a free choice of the backend, there is no relation to features `i32` or `u32`.